adapter/ipListChecker/cfg.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package ipListChecker;
+
+message Config {
+    // provider_url is the URL where to find the list to check against
+    string provider_url = 1;
+
+    // refresh_interval determines how often the provider is polled for
+    // an updated list unit: seconds
+    int64 refresh_interval = 2;
+
+    // TimeToLive indicates how long to keep a list before discarding it.
+    // Typically, the TTL value should be set to noticeably longer (> 2x) than the
+    // refresh interval to ensure continued operation in the face of transient
+    // server outages.
+    // unit: seconds
```

Comment:
I don't think we need nanos here. It is more natural to express time as seconds (or another unit) in a json or yaml file.